.eslintrc.js

```diff
@@ -1,3 +1,6 @@
 module.exports = {
   extends: ["@carforyou/eslint-config"],
+  rules: {
+    "jest/no-deprecated-functions": "off",
```

Comment:
What's the reason to disable it can't we update the deprecated functions we use?

Reply:
We don't use jest here and this one infers the jest version from the package version.